gemdaqmonitor/src/common/DaqMonitor.cc

```diff
@@ -19,8 +20,8 @@ gem::daqmon::DaqMonitor::DaqMonitor(const std::string& board_domain_name,log4cpl
 {
   DEBUG("DaqMonitor::DaqMonitor:: entering constructor");
   if (isConnected) { //TODO Add to the app monitoring space? Need to know in order to mask in web interface the boards which failed to connect
-    this->loadModule("amc", "amc v1.0.1");
-    DEBUG("DaqMonitor::DaqMonitor:: amc module loaded");
+    this->loadModule("daq_monitor", "daq_monitor v1.0.1");
```

Comment:
how important is this version hardcoding, and can it be removed in favour of some better way, i.e., a compile time variable that can be driven by the make process (or even extracted from the linked library)?